Packs/Base/Scripts/GetIndicatorsByQuery/GetIndicatorsByQuery.yml

```diff
@@ -52,7 +52,7 @@ commonfields:
   id: GetIndicatorsByQuery
   version: -1
 enabled: true
-name: GetIndicatorsByQuery
+name: HashIndicatorsFields
```

Comment:
Id and name should be the same - this will fail validation.
If you will change the id, it will be a BC break and require force merge (which I can't do)... are we sure this is needed?

Reply:
I know it should be the same, but @Nitzperetz asked me to change the script name, because it's too generic.
@Itay4 what we usually do in this case?